src/main/java/com/d2fn/guano/DumpJob.java

```diff
@@ -92,6 +96,18 @@ private void writeZnode(ZooKeeper zk, String outFile, String znode) throws Excep
                 out.flush();
                 out.close();
             }
+        } else {
+            if (!outFile.contains("_znode") && stat.getEphemeralOwner() == 0) {
+                // Create an empty folder for Ephemeral nodes.
+                File f = new File(outFile);
```

Comment:
Ideally I'd like this extracted into a little private method as it is an exact duplicate of the code in dump child except for the word "Empty" but all folders are created empty so I'd argue it's fine to use the same message for both cases.